src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
In type Props you have a type - Movie[]; it means that you will have array value in all cases
so, it's extra
```suggestion
export const MoviesList: React.FC<Props> = ({ movies }) => (
```